cpp/src/tests/test_utils.hpp

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2019, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <cudf/utilities/type_dispatcher.hpp>
+#include <cudf/utilities/traits.hpp>
+#include <rmm/device_buffer.hpp>
+
+#include <cudf/column/column.hpp>
+
+#include <thrust/distance.h>
+
+namespace detail {
+
+template <typename Element, typename InputIterator>
```

Comment:
I'm thinking this might be better to to put these utilities directly in the graph_test.cu file and having that file be the ONLY place where we test with cudf::column.